storybook/pages/DAppsWorkflowPage.qml

```diff
@@ -378,6 +379,19 @@ Item {
                 return Constants.TransactionEstimatedTime.LessThanThreeMins
             }
 
+            function getSuggestedFees() {
```

Comment:
Is this function intentionally put here? It looks like it will "hide" one from DAppsStore:
`   // Returns nim's SuggestedFeesDto; see src/app_service/service/transaction/dto.nim`
`   // Returns all value initialized to 0 if error`
`   function getSuggestedFees(chainId) {`
`       return JSON.parse(controller.getSuggestedFeesJson(chainId))`
`   }`